subsys/infinispan/src/main/java/org/commonjava/indy/subsys/infinispan/CacheProducer.java

```diff
@@ -67,10 +66,9 @@ protected CacheProducer()
     {
     }
 
-    public CacheProducer( IndyConfiguration indyConfiguration, EmbeddedCacheManager cacheManager )
+    public CacheProducer( IndyConfiguration indyConfiguration )
```

Comment:
How is the cache manager set? It doesn't look like it's @Inject-able?

Reply:
The cache manager is created in the start() -> defineCaches(). We don't need to inject it.